plugins/pubIds/doi/DOIPubIdPlugin.inc.php

```diff
@@ -18,6 +18,18 @@
 
 class DOIPubIdPlugin extends PubIdPlugin {
 
+	/**
+	 * @copydoc Plugin::register()
+	 */
+	public function register($category, $path) {
+		$success = parent::register($category, $path);
+		if (!Config::getVar('general', 'installed') || defined('RUNNING_UPGRADE')) return $success;
+		if ($success && $this->getEnabled()) {
+			HookRegistry::register('CitationStyleLanguage::citation', array($this, 'getCitationData'));
```

Comment:
Where is this hook called? (Is there some uncommitted code?)

Reply:
(Ah, found it: https://github.com/pkp/citationStyleLanguage/blob/master/CitationStyleLanguagePlugin.inc.php#L354)